### DISCUSSION TITLE
Getting 400 when calling GetSubmissions operation

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#879] [Q&A] [answered]
```
        data = {
        "sandbox": [
        {"action_script": "default",
            "command_line": "string",
            "document_password": "string",
            "enable_tor": False,
            "environment_id": 160,
            "network_settings": "string",
            "sha256": HASH
            }
        ]
        }
        x=  falconpy.APIHarness(creds={    "client_id": "_",    "client_secret": "__"})
        print(x.command("GetSubmissions", ids=x.command("Submit",data)))
```        
i got 400.

**Comment by jshcodes** [comment, marked as answer]
The code above will result in the full JSON return from the __`Submit`__ operation being provided to the `ids` keyword for the __`GetSubmissions`__ method. This will produce an error.

Instead, the `ids` keyword should be provided the contents of the _resources_ branch from the __`Submit`__ operation's JSON response. (Which in this case would be a list.) 

More information on response handling can be found [here](https://www.falconpy.io/Usage/Response-Handling.html).

#### Example
```python
print(x.command("GetSubmissions", ids=x.command("Submit",data)["body"]["resources"]))
```

> Please note: The contents of _**data**_ could also potentially generate errors when provided to the __`Submit`__ operation. If you are not using the value, you should not include it in the definition of this dictionary.
